src/styles/global.css

```
@@ -2,24 +2,35 @@
-  --ink-border: #5c4300;
+  --ink-brand-on-light: #c89200;
+  --ink-on-brand: #000000;
+  --ink-audience: #8432ff;
+  --ink-audience-hovered: #9b57ff;
+  --ink-on-audience: #ffffff;
+  --ink-border: #292c30;
   --ink-border-strong: #ffb300;
   --ink-focus: #ffd54f;
+  --ink-inverse: #ffffff;
+  --ink-on-inverse: #000000;
   --ink-font-heading: "Manrope Variable", Manrope, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Manrope" to be "manrope" (value-keyword-case)

(value-keyword-case)